debezium-server-voyager/yb-client-cdc-stream-wrapper/src/main/java/Main.java

```diff
@@ -51,8 +53,15 @@ public static void main(String[] args) throws Exception {
             masterAddressesList = masterAddressesList.replace("{", ""); //removing {}
             masterAddressesList = masterAddressesList.replace("}", "");
             System.out.println("Master Addresses: " + masterAddressesList);
+        } else if (parameters.getNumOfCDCStreams) {
+            ListCDCStreamsResponse cdcStreamsResponse = client.listCDCStreams(null, null, null);
```

Comment:
what are the parameters to this function? (to which we are passing NULLs)?

Reply:
got it, thanks! 
Shouldn't we be only checking for cdc streams in that database to which we are ingesting? 
Also, can we have streams on CQL tables? That is also something we shouldn't be concerned about, right?

Reply:
as discussed, will create ticket for this and will do it separately